cypress/integration/WebInterface/CQL Library/VersionAndDraftCQLLibrary.spec.ts

```diff
@@ -9,18 +9,14 @@ let updatedCqlLibraryName = ''
 
 describe('Add Version and Draft to CQL Library', () => {
 
-    before('Create CQL Library', () => {
-
+    beforeEach('Craete CQL Library and Login', () => {
         //Create CQL Library with Regular User
         CqlLibraryOne = 'TestLibrary1' + Date.now()
         CQLLibraryPage.createCQLLibraryAPI(CqlLibraryOne)
-
+        
         //Create Measure with Alternate User
         CqlLibraryTwo = 'TestLibrary2' + Date.now()
         CQLLibraryPage.createCQLLibraryAPI(CqlLibraryTwo, true, true)
```

Comment:
is there a reason you need to create a second library for 'Add Version to the CQL Library'?